cmd/autoheal/server.go

```diff
@@ -70,6 +71,10 @@ func init() {
 			"directory all the files inside whose names end in .yml or .yaml will be "+
 			"loaded, in alphabetical order.",
 	)
+	serverFlags.BoolVar(
+		&serverWatchConfigFiles, "watch-config-files", false,
+		"Watch config files for changes.",
+	)
```

Comment:
I think this should be always `true` and not configurable.

Reply:
ok :-)

@jhernand @nimrodshn if you are here :-) one Q:

Should we delete all items in `h.rulesCache` each time we reload new version of rules, and re-populate the cache list ?

Reply:
Yes, when the configuration has changed we should remove from the cache all the rules that were loaded from that configuration, and then load them again.